doc/orbits.xml

```diff
@@ -123,21 +138,27 @@ gap> 5 / gens[sch[5]];
 <ManSection>
   <Attr Name="DigraphOrbitReps" Arg="digraph"/>
   <Returns>
-    A list of integers.
+    An immutable list of integers.
   </Returns>
```

Comment:
Wasn't there some discussion about returning mutable lists if that was an option, because mutable lists can always be made immutable if desirable, and but an immutable list has to be copied to be made mutable?

Reply:
This is an attribute though, so the value has to be mutable.